packages/schematics/angular/workspace/files/tslint.json.template

```diff
@@ -61,15 +61,22 @@
       "single"
     ],
     "trailing-comma": false,
-    "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "banana-in-box": true,
+    "component-class-suffix": true,
+    "contextual-life-cycle": true,
+    "directive-class-suffix": true,
+    "no-attribute-parameter-decorator": true,
```

Comment:
We can drop this. The attribute decorator might not be the best practice but we can keep it.